### PR TITLE
Create wallabag2 documentation page - EN

### PR DESCRIPTION
**Pull request by lapineige**

### CODE REVIEW

File: userdoc.md [hunk 1]
```diff
@@ -10,6 +10,6 @@
    * [Tiny Tiny RSS](/app_ttrss_en)
    * [Transmission](/app_transmission_en)
    * [Radicale](/app_radicale)
-   * [RoundCube](/app_roundcube_en)
```
Comment (M5oul) [inline userdoc.md]: Why did you remove Roundcube link?
[The page exist](https://yunohost.org/#/app_roundcube).

Comment (lapineige) [inline userdoc.md]: Sorry, I made a mistake.
Fixed.
